Make.rules

```diff
@@ -64,7 +64,7 @@ export FULL_VERSION RCS_FIND_IGNORE
 # Utilities and default flags for them.
 
 CROSS_COMPILE	?=
-CC		:= $(CROSS_COMPILE)gcc
+CC		:= $(CROSS_COMPILE)cc
```

Comment:
LGTM :+1:

You can also override CC when you run make: `make CC=cc`

Reply:
CC cannot be overridden. The line here is:
```
CC		:= $(CROSS_COMPILE)cc
```
To be able to override CC, the line should be:
```
CC		?= $(CROSS_COMPILE)cc
```

Reply:
The `?=` is to set if there's no other definition anywhere. Command line and environment overrides `=` and `:=` too. To have makefile assignment override those you need to use `override` directive. See: https://www.gnu.org/software/make/manual/html_node/Override-Directive.html

Reply:
Hmm... `$(CROSS_COMPILE)cc` seems to be not defined in Alpine. Should that be added? But as such this will break cross-compile.